packages/nimble-components/src/select/index.ts

```diff
@@ -837,6 +944,23 @@ export class Select
         this.committedSelectedOption = options[this.selectedIndex];
     }
 
+    private toggleNewActiveOption(getNewActiveIndex: () => number): void {
```

Comment:
The behavior of setting `aria-selected` on the active option seems incorrect to me, though I will admit that I haven't studied all the ARIA guidelines as much as I'm sure you have.

However, if the active option isn't considered the selected option until it is "committed", then shouldn't `aria-selected` follow that? That would mean that the value of the `aria-selected` option should always be in sync with the `value` and `selectedIndex` of the select. That also seems to align with the [Select-Only Combobox Example](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/).